airflow/contrib/operators/gcp_bigtable_operator.py

```diff
@@ -0,0 +1,376 @@
+        super(BigTableInstanceDeleteOperator, self).__init__(*args, **kwargs)
+
+    def execute(self, context):
+        try:
+            self.hook.delete_instance(self.project_id, self.instance_id)
+        except google.api_core.exceptions.NotFound:
+            self.log.info("The Instance '{}' does not exist in project '{}'.".format(self.instance_id, self.project_id))
+        except google.api_core.exceptions.GoogleAPICallError as e:
+            self.log.error('An error occurred. Exiting.')
+            raise e
+
+
+class BigTableTableCreateOperator(BaseOperator, BigTableValidationMixin):
+    """
+    Creates the table in BigTable instance.
+
+    For more details about creating table have a look at the reference:
+    https://googleapis.github.io/google-cloud-python/latest/bigtable/table.html#google.cloud.bigtable.table.Table.create
+
+    :type project_id: str
+    :param project_id: The Google Developers Console [project ID or project number]
+    :type instance_id: str
+    :param instance_id: The ID of the instance that will hold the new Table
+    :type table_id: str
+    :param table_id: The ID of the Table to be created
+    """
+    REQUIRED_ATTRIBUTES = ('project_id', 'instance_id', 'table_id')
+    template_fields = ['project_id', 'instance_id', 'table_id']
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 instance_id,
+                 table_id,
+                 initial_split_keys=None,
+                 column_families=None,
+                 *args, **kwargs):
+        self.project_id = project_id
+        self.instance_id = instance_id
+        self.table_id = table_id
+        self.initial_split_keys = initial_split_keys or list()
+        self.column_families = column_families or dict()
+        self._validate_inputs()
+        self.hook = BigTableHook()
+        self.instance = None
+        super(BigTableTableCreateOperator, self).__init__(*args, **kwargs)
+
+    def _compare_column_families(self):
+        table_column_families = self.hook.get_column_families_for_table(self.instance, self.table_id)
+        if table_column_families.keys() != self.column_families.keys():
+            self.log.error("Table '{}' has different set of Column Families".format(self.table_id))
+            self.log.error("Expected: {}".format(self.column_families.keys()))
+            self.log.error("Actual: {}".format(table_column_families.keys()))
+            return False
+
+        for key in table_column_families.keys():
+            if table_column_families[key].gc_rule != self.column_families[key]:
+                self.log.error("Column Family '{}' differs for Table '{}'.".format(key, self.table_id))
+                return False
+        return True
+
+    def execute(self, context):
+        self.instance = self.hook.get_instance(self.project_id, self.instance_id)
+        if not self.instance:
+            raise AttributeError("The Instance '{}' does not exist in project '{}'.".format(
```

Comment:
Why not raise AirflowException? We are doing it usually in operators as it will get converted to Airflow Exception later if it's not one. Most of operators does this from what I saw. Similar everywhere.